test/parallel/test-process-binding.js

```diff
@@ -6,7 +6,7 @@ const { internalBinding } = require('internal/test/binding');
 
 assert.throws(
   function() {
-    process.binding('test');
+    internalBinding('test');
```

Comment:
This change is unnecessary since this file is specifically testing `process.binding`